go/plugins/compat_oai/generate_live_test.go

```diff
@@ -111,68 +112,77 @@ func TestGenerator_Stream(t *testing.T) {
 }
 
 func TestWithConfig(t *testing.T) {
```

Comment:
@kekoawong 

The `TestWithConfig` doesn't rely on an API key, so it doesn't need to be a live test.
We could move `TestWithConfig` to a separate test file, like generate_test.go.
What do you think?

Reply:
When we get a result we still use the generate function so will keep it in here for now!

`result, err := generator.WithMessages(messages).WithConfig(tt.config).Generate(context.Background(), nil)`

Reply:
👍